docs/src/manual/models.md

```diff
@@ -247,6 +247,21 @@ julia> time_limit_sec(model)
 Inf
 ```
 
+To avoid making unit-conversion mistakes with [`set_time_limit_sec`](@ref),
+consider converting a `Dates.Period` object to seconds:
+
+```jldoctest time_limit
+julia> import Dates
+
+julia> seconds(x::Dates.Period) = 1e-9 * Dates.value(floor(x, Dates.Nanosecond))
```

Comment:
Instead of using a magic constant (which is a reason why I brought the topic to the table), you could do something like `convert(Second, Hour(72))` or `Second(Hour(72))`. But it may have problems if the number is not an integer number of seconds.

```julia
julia> convert(Dates.Minute, Dates.Millisecond(75631000))
ERROR: InexactError: divexact(Int64, 1260.5166666666667)
Stacktrace:
 [1] divexact
   @ C:\Users\Thibaut\AppData\Local\Programs\Julia-1.8.2\share\julia\stdlib\v1.8\Dates\src\periods.jl:414 [inlined]
 [2] convert(#unused#::Type{Minute}, x::Millisecond)
   @ Dates C:\Users\Thibaut\AppData\Local\Programs\Julia-1.8.2\share\julia\stdlib\v1.8\Dates\src\periods.jl:441
 [3] top-level scope
   @ REPL[61]:1
```

Does it make sense to support sub-second resolution? Are solvers precise enough for this?

Reply:
I guess you could use `seconds(x::Dates.Period) = Dates.value(ceil(x, Dates.Second))`

Reply:
> Does it make sense to support sub-second resolution? Are solvers precise enough for this?

Absolutely, we shouldn't arbitrarily round to seconds. I'd guess milliseconds would be fine if we don't want to do nanoseconds. We regularly get questions about solving problems in a millisecond time frame.

Reply:
Updated. I've never seen anyone measure a time in in microseconds.